scripts/create_gitlab_snapshot.py

```diff
@@ -67,6 +83,17 @@ def shutdown_instance(instance: Mapping[str, JSON]):
     log.info('Instance %s has stopped', instance_ids)
 
 
+def restart_instance(instance: Mapping[str, JSON]):
+    instance_ids = [instance['InstanceId']]
+    log.info('Restarting %s  GitLab instance', config.deployment_stage)
+    aws.ec2.start_instances(InstanceIds=instance_ids)
+    log.info('Waiting for the GitLab instance to start')
+    waiter = aws.ec2.get_waiter('instance_running')
+    waiter.wait(InstanceIds=instance_ids,
+                WaiterConfig=dict(MaxAttempts=9999, Delay=15))
```

Comment:
IIRC, there are other waiters in this module. I don't think they use astronomical values like this. Be sure to test the timeout case.

Reply:
[In past feedback](https://github.com/DataBiosphere/azul/pull/3627#discussion_r814141380), regarding waiting the recommendation was "It should wait forever". But perhaps for starting the instance over we should wait the default 10 min?

Reply:
@hannes-ucsc: "I don't remember correctly, this is consistent with precedent and should stay as is."